plugins/Feedback/templates/index.twig

```diff
@@ -23,12 +23,12 @@
 
             <h4>{{'Feedback_PopularHelpTopics'|translate}}</h4>
             {% for helpTopic in popularHelpTopics %}
-                <p> &bull;  <a rel="noreferrer noopener" target="_blank" href="{{helpTopic.url|e('html_attr')}}">{{helpTopic.title|translate}}</a></p>
+                <p> &bull;  <a rel="noreferrer noopener" target="_blank" href="{{ helpLink(helpTopic.url) }}">{{helpTopic.title|translate}}</a></p>
```

Comment:
Here and for the links below it would be good to keep `|e('html_attr')` instead of `|raw`. This ensures that the html wouldn't break if the returned link would contain a character like `"`